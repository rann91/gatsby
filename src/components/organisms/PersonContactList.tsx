/* @jsx jsx */
import { Box, Container, Flex, jsx } from 'theme-ui'
import { column } from '../../gatsby-plugin-theme-ui'
import {
  Strapi_ComponentContentPersonContactList,
  Strapi_Person
} from '../../typings/graphql'
import Section from '../atoms/Section'
import PersonContactTeaser from '../molecules/PersonContactTeaser'
import HeadingGroup from '../molecules/HeadingGroup'

const PersonContactList = ({
  title,
  subtitle,
  persons
}: Strapi_ComponentContentPersonContactList) => (
  <Section data-testid="person-contact-list" hasTitle={!!title}>
    <Container>
      <HeadingGroup {...{ title, subtitle }} />
      <Flex
        sx={{
          flexWrap: 'wrap',
          '* + &': {
            paddingTop: [0, null, null, 5]
          }
        }}>
        {((persons || []) as Strapi_Person[]).map(person => (
          <Box
            key={person.id}
            p={5}
            sx={{
              width: [column(12), column(6), null, column(4)],
              textAlign: ['center', null, null, 'left']
            }}>
            <PersonContactTeaser {...person} />
          </Box>
        ))}
      </Flex>
    </Container>
  </Section>
)

export default PersonContactList
