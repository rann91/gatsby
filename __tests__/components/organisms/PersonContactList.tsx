import { render } from '@testing-library/react'
import React from 'react'
import PersonContactList from '../../../src/components/organisms/PersonContactList'
import component from '../../fixtures/component-person-contact-list.json'

describe('PersonContactList', () => {
  it('renders correctly', () => {
    const { container } = render(<PersonContactList {...component} />)

    expect(container).toMatchSnapshot()
  })
})
