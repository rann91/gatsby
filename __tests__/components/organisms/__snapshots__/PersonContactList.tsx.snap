// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonContactList renders correctly 1`] = `
<div>
  <section
    class="css-1ptauik"
    data-testid="person-contact-list"
  >
    <div
      class="css-1yrmf69"
    >
      <div
        class="css-1mpaekt-HeadingGroup"
      >
        <div
          class="css-146vepc-HeadingGroup"
          data-testid="subtitle"
        >
          What we are best at
        </div>
        <h2
          class="css-16nj6cn-HeadingGroup"
          data-testid="title"
        >
          Our services
        </h2>
      </div>
      <div
        class="css-89nsph-PersonContactList"
      >
        <div
          class="css-itco2b"
        >
          <div
            class="css-vurnku"
          >
            <div
              class="css-i9l3dq"
            >
              <div
                class="css-9cwazt"
              />
              <div
                class="css-1sr1905"
              >
                <div
                  class=" gatsby-image-wrapper"
                  style="position: relative; overflow: hidden;"
                >
                  <div
                    aria-hidden="true"
                    style="width: 100%; padding-bottom: 123.75%;"
                  />
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAZABQDASIAAhEBAxEB/8QAGAABAAMBAAAAAAAAAAAAAAAAAAMEBQb/xAAWAQEBAQAAAAAAAAAAAAAAAAACAQD/2gAMAwEAAhADEAAAAYVfQ1IlLPvUkegTiv/EAB8QAAEDBAMBAAAAAAAAAAAAAAIAAQMEERIhEyIxMv/aAAgBAQABBQKQUwDjHCxjvG7WbSqOsS4QNVHii+P/xAAWEQADAAAAAAAAAAAAAAAAAAABESD/2gAIAQMBAT8BCj//xAAVEQEBAAAAAAAAAAAAAAAAAAAgIf/aAAgBAgEBPwGj/8QAHhAAAgEEAwEAAAAAAAAAAAAAAAERAhAhMRJBYXH/2gAIAQEABj8CWTTkmEJ9G0OW1kpp9tygo+X/AP/EAB0QAQACAgIDAAAAAAAAAAAAAAEAESExQWEQodH/2gAIAQEAAT8hJ+MRmAecjM1GalDvy+4JTK+4RBDhADdoLq8alUStOGvFG5rn/9oADAMBAAIAAwAAABCgHc3/xAAXEQADAQAAAAAAAAAAAAAAAAAAAREh/9oACAEDAQE/EKPSjIj/xAAYEQEBAAMAAAAAAAAAAAAAAAABABEhMf/aAAgBAgEBPxBBsLDE9v/EACAQAQACAQQCAwAAAAAAAAAAAAEAETEhQXGhUbFhkdH/2gAIAQEAAT8Q3hNdMCZBmtAK8GI6zPCgrY0ipjQtObLXNQmUQStEJeU65gbCrpiDfrb3/YsLKH0llYgr2o3+Z0PomLk9zsT/2Q=="
                    style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: 50% -5rem; opacity: 1; transition-delay: 500ms;"
                  />
                  <picture>
                    <source
                      sizes="(max-width: 640px) 100vw, 640px"
                      srcset="/static/75e03c0319787e05c861002a4eab7484/e75b5/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 160w,
/static/75e03c0319787e05c861002a4eab7484/c01e2/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 320w,
/static/75e03c0319787e05c861002a4eab7484/f422e/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 640w,
/static/75e03c0319787e05c861002a4eab7484/a6352/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 960w,
/static/75e03c0319787e05c861002a4eab7484/ec6c5/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 1280w,
/static/75e03c0319787e05c861002a4eab7484/a6974/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 1536w"
                    />
                    <img
                      alt=""
                      loading="lazy"
                      sizes="(max-width: 640px) 100vw, 640px"
                      src="/static/75e03c0319787e05c861002a4eab7484/f422e/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg"
                      srcset="/static/75e03c0319787e05c861002a4eab7484/e75b5/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 160w,
/static/75e03c0319787e05c861002a4eab7484/c01e2/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 320w,
/static/75e03c0319787e05c861002a4eab7484/f422e/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 640w,
/static/75e03c0319787e05c861002a4eab7484/a6352/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 960w,
/static/75e03c0319787e05c861002a4eab7484/ec6c5/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 1280w,
/static/75e03c0319787e05c861002a4eab7484/a6974/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 1536w"
                      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: 50% -5rem; opacity: 0; transition: opacity 500ms;"
                    />
                  </picture>
                  <noscript>
                    &lt;picture&gt;&lt;source srcset="/static/75e03c0319787e05c861002a4eab7484/e75b5/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 160w,
/static/75e03c0319787e05c861002a4eab7484/c01e2/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 320w,
/static/75e03c0319787e05c861002a4eab7484/f422e/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 640w,
/static/75e03c0319787e05c861002a4eab7484/a6352/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 960w,
/static/75e03c0319787e05c861002a4eab7484/ec6c5/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 1280w,
/static/75e03c0319787e05c861002a4eab7484/a6974/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 1536w" sizes="(max-width: 640px) 100vw, 640px" /&gt;&lt;img loading="lazy" sizes="(max-width: 640px) 100vw, 640px" srcset="/static/75e03c0319787e05c861002a4eab7484/e75b5/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 160w,
/static/75e03c0319787e05c861002a4eab7484/c01e2/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 320w,
/static/75e03c0319787e05c861002a4eab7484/f422e/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 640w,
/static/75e03c0319787e05c861002a4eab7484/a6352/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 960w,
/static/75e03c0319787e05c861002a4eab7484/ec6c5/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 1280w,
/static/75e03c0319787e05c861002a4eab7484/a6974/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg 1536w" src="/static/75e03c0319787e05c861002a4eab7484/f422e/mubariz_mehdizadeh_t3zr_Em88ehc_unsplash_4a854337d9.jpg" alt="" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
                  </noscript>
                </div>
              </div>
            </div>
          </div>
          <h3
            class="css-zmc4ii-PersonContactTeaser"
          >
            Michael Fassbender
          </h3>
          <p
            class="css-1uw231t-PersonContactTeaser"
          >
            CEO
          </p>
          <hr
            class="css-17tuus4"
          />
          <div
            class="css-rp6q1l-PersonContactTeaser"
          >
            <p
              class="css-vurnku"
            >
              <a
                class="css-1audd3q-PersonContactTeaser"
                href="mailto:ceo@business.com"
              >
                ceo@business.com
              </a>
            </p>
            <p
              class="css-vurnku"
            >
              01 (650) 658 6823
            </p>
          </div>
        </div>
        <div
          class="css-itco2b"
        >
          <div
            class="css-vurnku"
          >
            <div
              class="css-i9l3dq"
            >
              <div
                class="css-9cwazt"
              />
              <div
                class="css-1sr1905"
              >
                <div
                  class=" gatsby-image-wrapper"
                  style="position: relative; overflow: hidden;"
                >
                  <div
                    aria-hidden="true"
                    style="width: 100%; padding-bottom: 123.75%;"
                  />
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAZABQDASIAAhEBAxEB/8QAGQAAAgMBAAAAAAAAAAAAAAAAAAQBAgMF/8QAFgEBAQEAAAAAAAAAAAAAAAAAAwIE/9oADAMBAAIQAxAAAAHKbZlOZQoXH1GI28oqIH//xAAeEAABAwUBAQAAAAAAAAAAAAABAgMRAAQSIzMhMv/aAAgBAQABBQJsZlaSCYpjxspK6mKZTpjBFyNrXA/Nz1//xAAbEQACAQUAAAAAAAAAAAAAAAAAAgEDECEiMf/aAAgBAwEBPwGKetl4MuT/xAAYEQACAwAAAAAAAAAAAAAAAAAAARARQf/aAAgBAgEBPwHSoR//xAAdEAABBAIDAAAAAAAAAAAAAAAAAQIQISJxMUGB/9oACAEBAAY/AjmHdmKXGyjaDBTw/8QAHRAAAgICAwEAAAAAAAAAAAAAAREAMSFBEFGhkf/aAAgBAQABPyE5galbLRcISsy9yf2C7BzAQI3tio5+7EXvQ+VIM8M8nE//2gAMAwEAAgADAAAAENcvcf/EABgRAQEBAQEAAAAAAAAAAAAAAAEAESFB/9oACAEDAQE/EAd+2JyGDIFt/8QAGhEAAgIDAAAAAAAAAAAAAAAAAAEQESExQf/aAAgBAgEBPxC9IdY2Ef/EACAQAQACAQMFAQAAAAAAAAAAAAEAESExUWFBcaGxwYH/2gAIAQEAAT8QXagZU0PsLub0FvRuYQQaI1edo2kwqL2Y/IQGsDALx7m5m0e1ekS9hWgrj5Aum5qErVIcul+J4Sebml2e2f/Z"
                    style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: 50% -5rem; opacity: 1; transition-delay: 500ms;"
                  />
                  <picture>
                    <source
                      sizes="(max-width: 640px) 100vw, 640px"
                      srcset="/static/787145d790893b6c6472d451ec5ff8da/e75b5/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 160w,
/static/787145d790893b6c6472d451ec5ff8da/c01e2/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 320w,
/static/787145d790893b6c6472d451ec5ff8da/f422e/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 640w,
/static/787145d790893b6c6472d451ec5ff8da/a6352/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 960w,
/static/787145d790893b6c6472d451ec5ff8da/ec6c5/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 1280w,
/static/787145d790893b6c6472d451ec5ff8da/a6974/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 1536w"
                    />
                    <img
                      alt=""
                      loading="lazy"
                      sizes="(max-width: 640px) 100vw, 640px"
                      src="/static/787145d790893b6c6472d451ec5ff8da/f422e/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg"
                      srcset="/static/787145d790893b6c6472d451ec5ff8da/e75b5/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 160w,
/static/787145d790893b6c6472d451ec5ff8da/c01e2/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 320w,
/static/787145d790893b6c6472d451ec5ff8da/f422e/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 640w,
/static/787145d790893b6c6472d451ec5ff8da/a6352/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 960w,
/static/787145d790893b6c6472d451ec5ff8da/ec6c5/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 1280w,
/static/787145d790893b6c6472d451ec5ff8da/a6974/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 1536w"
                      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: 50% -5rem; opacity: 0; transition: opacity 500ms;"
                    />
                  </picture>
                  <noscript>
                    &lt;picture&gt;&lt;source srcset="/static/787145d790893b6c6472d451ec5ff8da/e75b5/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 160w,
/static/787145d790893b6c6472d451ec5ff8da/c01e2/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 320w,
/static/787145d790893b6c6472d451ec5ff8da/f422e/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 640w,
/static/787145d790893b6c6472d451ec5ff8da/a6352/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 960w,
/static/787145d790893b6c6472d451ec5ff8da/ec6c5/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 1280w,
/static/787145d790893b6c6472d451ec5ff8da/a6974/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 1536w" sizes="(max-width: 640px) 100vw, 640px" /&gt;&lt;img loading="lazy" sizes="(max-width: 640px) 100vw, 640px" srcset="/static/787145d790893b6c6472d451ec5ff8da/e75b5/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 160w,
/static/787145d790893b6c6472d451ec5ff8da/c01e2/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 320w,
/static/787145d790893b6c6472d451ec5ff8da/f422e/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 640w,
/static/787145d790893b6c6472d451ec5ff8da/a6352/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 960w,
/static/787145d790893b6c6472d451ec5ff8da/ec6c5/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 1280w,
/static/787145d790893b6c6472d451ec5ff8da/a6974/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg 1536w" src="/static/787145d790893b6c6472d451ec5ff8da/f422e/max_ilienerwise_Yv_WJOXHNJ_94_unsplash_431f0f55d0.jpg" alt="" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
                  </noscript>
                </div>
              </div>
            </div>
          </div>
          <h3
            class="css-zmc4ii-PersonContactTeaser"
          >
            Amanda Peterson
          </h3>
          <p
            class="css-1uw231t-PersonContactTeaser"
          >
            Design Director
          </p>
          <hr
            class="css-17tuus4"
          />
          <div
            class="css-rp6q1l-PersonContactTeaser"
          >
            <p
              class="css-vurnku"
            >
              <a
                class="css-1audd3q-PersonContactTeaser"
                href="mailto:design@business.com"
              >
                design@business.com
              </a>
            </p>
            <p
              class="css-vurnku"
            >
              01 (650) 658 6825
            </p>
          </div>
        </div>
        <div
          class="css-itco2b"
        >
          <div
            class="css-vurnku"
          >
            <div
              class="css-i9l3dq"
            >
              <div
                class="css-9cwazt"
              />
              <div
                class="css-1sr1905"
              >
                <div
                  class=" gatsby-image-wrapper"
                  style="position: relative; overflow: hidden;"
                >
                  <div
                    aria-hidden="true"
                    style="width: 100%; padding-bottom: 123.75%;"
                  />
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAZABQDASIAAhEBAxEB/8QAGAAAAwEBAAAAAAAAAAAAAAAAAAIEAwX/xAAWAQEBAQAAAAAAAAAAAAAAAAACAQD/2gAMAwEAAhADEAAAAak5zBVHJEaMHkU0KSb/xAAeEAABBAEFAAAAAAAAAAAAAAADAAECERIhIjIzQf/aAAgBAQABBQJi1KR1E+lopGu4qW0mTOsbRuzxuP8A/8QAFhEBAQEAAAAAAAAAAAAAAAAAABEB/9oACAEDAQE/AarUf//EABYRAQEBAAAAAAAAAAAAAAAAAAABEf/aAAgBAgEBPwGRiNf/xAAfEAACAQMFAQAAAAAAAAAAAAAAAREQMXEhIkGBkaH/2gAIAQEABj8CHtamxf6T6RY1k6E3xRYosH//xAAfEAACAgIBBQAAAAAAAAAAAAABEQAhMUEQUWGBocH/2gAIAQEAAT8hRk6yYYGqGx3Ov+EjMEKG0MQGy4ilntcJIKV7htQolCSEVPRT64L/2gAMAwEAAgADAAAAEO8qPv/EABcRAQEBAQAAAAAAAAAAAAAAAAEAESH/2gAIAQMBAT8QQi25dyxf/8QAGREAAgMBAAAAAAAAAAAAAAAAAAEQESEx/9oACAECAQE/EGcimFj/xAAeEAEAAgIDAAMAAAAAAAAAAAABESEAMUFRYRChwf/aAAgBAQABPxATQA6COchFUpoN6Gnm8hsEqxAPIjCKTqax1kLyaAiHpP3GEgd1D7gK0VyRN6dsVj0Ip6lNM9Xk6gIU/GHbH12f/9k="
                    style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: 50% -5rem; opacity: 1; transition-delay: 500ms;"
                  />
                  <picture>
                    <source
                      sizes="(max-width: 640px) 100vw, 640px"
                      srcset="/static/d570669882a564e99405b9ad24d15999/e75b5/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 160w,
/static/d570669882a564e99405b9ad24d15999/c01e2/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 320w,
/static/d570669882a564e99405b9ad24d15999/f422e/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 640w,
/static/d570669882a564e99405b9ad24d15999/a6352/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 960w,
/static/d570669882a564e99405b9ad24d15999/ec6c5/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 1280w,
/static/d570669882a564e99405b9ad24d15999/a6974/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 1536w"
                    />
                    <img
                      alt=""
                      loading="lazy"
                      sizes="(max-width: 640px) 100vw, 640px"
                      src="/static/d570669882a564e99405b9ad24d15999/f422e/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg"
                      srcset="/static/d570669882a564e99405b9ad24d15999/e75b5/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 160w,
/static/d570669882a564e99405b9ad24d15999/c01e2/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 320w,
/static/d570669882a564e99405b9ad24d15999/f422e/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 640w,
/static/d570669882a564e99405b9ad24d15999/a6352/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 960w,
/static/d570669882a564e99405b9ad24d15999/ec6c5/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 1280w,
/static/d570669882a564e99405b9ad24d15999/a6974/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 1536w"
                      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: 50% -5rem; opacity: 0; transition: opacity 500ms;"
                    />
                  </picture>
                  <noscript>
                    &lt;picture&gt;&lt;source srcset="/static/d570669882a564e99405b9ad24d15999/e75b5/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 160w,
/static/d570669882a564e99405b9ad24d15999/c01e2/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 320w,
/static/d570669882a564e99405b9ad24d15999/f422e/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 640w,
/static/d570669882a564e99405b9ad24d15999/a6352/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 960w,
/static/d570669882a564e99405b9ad24d15999/ec6c5/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 1280w,
/static/d570669882a564e99405b9ad24d15999/a6974/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 1536w" sizes="(max-width: 640px) 100vw, 640px" /&gt;&lt;img loading="lazy" sizes="(max-width: 640px) 100vw, 640px" srcset="/static/d570669882a564e99405b9ad24d15999/e75b5/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 160w,
/static/d570669882a564e99405b9ad24d15999/c01e2/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 320w,
/static/d570669882a564e99405b9ad24d15999/f422e/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 640w,
/static/d570669882a564e99405b9ad24d15999/a6352/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 960w,
/static/d570669882a564e99405b9ad24d15999/ec6c5/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 1280w,
/static/d570669882a564e99405b9ad24d15999/a6974/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg 1536w" src="/static/d570669882a564e99405b9ad24d15999/f422e/warren_wong_VVE_w_JJRRH_gk_unsplash_5b8e624c4f.jpg" alt="" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
                  </noscript>
                </div>
              </div>
            </div>
          </div>
          <h3
            class="css-zmc4ii-PersonContactTeaser"
          >
            Joshua Harris
          </h3>
          <p
            class="css-1uw231t-PersonContactTeaser"
          >
            CTO
          </p>
          <hr
            class="css-17tuus4"
          />
          <div
            class="css-rp6q1l-PersonContactTeaser"
          >
            <p
              class="css-vurnku"
            >
              <a
                class="css-1audd3q-PersonContactTeaser"
                href="mailto:cto@business.com"
              >
                cto@business.com
              </a>
            </p>
            <p
              class="css-vurnku"
            >
              01 (650) 658 6824
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
